node_groups.tf

```diff
@@ -115,6 +115,16 @@ locals {
       type        = "ingress"
       self        = true
     }
+    # metrics-server
+    ingress_cluster_8443_webhook = {
```

Comment:
@bryantbiggs 8443 is typo in the SG name!

Reply:
gosh dangit - thanks!

Reply:
#2354 should resolve - thanks for catching